webpack.config.js

```diff
@@ -97,6 +98,25 @@ const config = {
                 ],
             },
         }),
+        new CopyPlugin({
+            patterns: [
+                {
+                    from: path.resolve(__dirname, 'node_modules/aws-ssm-document-language-service/dist/server.js'),
+                    to: path.resolve(__dirname, 'dist/src/ssmDocument/ssm/ssmServer.js'),
+                },
+                {
+                    from: path.resolve(
+                        __dirname,
+                        'node_modules/aws-ssm-document-language-service/dist/server.js.LICENSE.txt'
+                    ),
+                    to: path.resolve(__dirname, 'dist/src/ssmDocument/ssm/ssmServer.js.LICENSE.txt'),
+                },
+                {
+                    from: path.resolve(__dirname, 'node_modules/aws-ssm-document-language-service/dist/server.js.map'),
+                    to: path.resolve(__dirname, 'dist/src/ssmDocument/ssm/ssmServer.js.map'),
+                },
+            ],
+        }),
```

Comment:
This is copying our language server bundle into the build. The `aws-ssm-document-language-service` package is bundling everything into a single js file. VSCode is starting the language server in a separate Node process, pointing to this file (see `ssmClient.ts`).